app-frontend/packs/store/bannerModule.js

```diff
@@ -0,0 +1,99 @@
+import axios from "axios"
+import router from "../router" 
+export const bannerModule = {
+    state() {
+        return {
+            bannersData: [],
+            wysiwygText: '',
+            wysiwygHtml: '',
+            bannerTitle: '',
+            bannerColor: '#FFFFF',
+            bannerProductId: Number,
+            bannerId: Number,
+            isLoader: false
+        }
+    },
+    mutations: {
+        setData(state, value) {
+            state.bannersData = value
+        },
+        setWysiwygText(state, value) {
+            state.wysiwygText = value
+        },
+        setWysiwygHtml(state, value) {
+            state.wysiwygHtml = value
+        },
+        setBannerTitle(state, value) {
+            state.bannerTitle = value
+        },
+        setBannerColor(state, value) {
+            state.bannerColor = value
+        },
+        setBannerProductId(state, value) {
+            state.bannerProductId = value
+        },
+        setBannerId(state, value) {
+            state.bannerId = value
+        }
+
+    },
+    actions: {
+        async fetchBanners({commit, state}) {
```

Comment:
either async await or promises, need to change everywhere

Reply:
+ change